src/vs/workbench/contrib/chat/browser/chatContentParts/toolInvocationParts/chatTerminalToolProgressPart.ts

```diff
@@ -130,6 +141,12 @@ class TerminalCommandDecoration extends Disposable {
 			}
 		}
 
+		if (!this._hover.value) {
+			this._hover.value = this._options.hoverService.setupManagedHover(getDefaultHoverDelegate('mouse'), decoration, this._currentHoverText);
+		} else {
+			this._hover.value.update(this._currentHoverText);
+		}
```

Comment:
You should never use `setupManagedHover` anymore, should be crossed out in vscode as it's deprecated. Use `setupDelayedHover` instead